project.clj

```diff
@@ -152,10 +143,11 @@
               [lein-doo "0.1.10"]]
     :repl-options {:init-ns user
                    :nrepl-middleware [cider.piggieback/wrap-cljs-repl]}}}
+
   :cljsbuild
   {:builds
    [{:id "dev-ui"
-     :source-paths ["src" "test" "dev/ui"]
+     :source-paths ["src/ethlance/ui" "src/ethlance/shared" "dev/ui"]
```

Comment:
what is the reason for going back to this?

Reply:
fixed